dts/bindings/bluetooth/nxp,bt-hci-uart.yaml

```diff
@@ -17,12 +17,22 @@ properties:
     type: phandle-array
     required: true
 
-  hci-operation-speed:
+  hci-default-speed:
     type: int
     description: |
-      HCI UART baudrate for feature operation. If not defined
+      HCI UART default baudrate after the FW download. If not defined
       bus/current-speed will be used as default.
 
+  hci-operation-speed:
+    type: int
+    description: |
+      HCI UART running baudrate for feature operation. If not defined
+      hci-default-speed will be used as default.
+      When hci-operation-speed is different with hci-default-speed,
+      the HCI UART baudrate update process will be involved to change
+      the HCI UART baudrate from hci-default-speed to
+      hci-operation-speed.
+
```

Comment:
I have a few questions about these.

1. If these are specific to this driver, then shouldn't you prefix them with `nxp,` (or possibly with `zephyr,` as per https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#the-zephyr-prefix)?
2. Does there perhaps exist some existing properties elsewhere with the same semantics, if so, consider reusing those instead of defining new ones.
3. Since the HCI node will be a child node of a UART device, could you derive some information of the baud rate from the UART node instead?

Reply:
> 1. If these are specific to this driver, then shouldn't you prefix them with `nxp,` (or possibly with `zephyr,` as per https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#the-zephyr-prefix)?

It is a member of `[nxp,bt-hci-uart.yaml](https://github.com/zephyrproject-rtos/zephyr/pull/82308/files#diff-1d64be33434b758bd8cc34a07c9a3d770c1f393fbf681ec1fb8914ab44f95ad8)`. And I do not think it is not a member defined by zephyr. But I did not notice it should be prefixed with `nxp,`. Since it is a member of `nxp,bt-hci-uart.yaml`. And file `nxp,bt-hci-uart.yaml` has been prefixed with `nxp`, is it necessary to add a prefix to each member of `nxp,bt-hci-uart.yaml`?

> 2. Does there perhaps exist some existing properties elsewhere with the same semantics, if so, consider reusing those instead of defining new ones.

Yes. `current-speed` of UART can be used to achieve the purpose. I have remove the changes of `hci-default-speed` from this PR.

> 3. Since the HCI node will be a child node of a UART device, could you derive some information of the baud rate from the UART node instead?

Yes. `current-speed` is used as the default bandrate for HCI.